test/integration/__snapshots__/lerna-publish.test.js.snap

```diff
@@ -83,33 +83,41 @@ Array [
 ]
 `;
 
-exports[`stdout: updates fixed versions 1`] = `
-"Lerna __TEST_VERSION__
-Current version: 1.0.0
-Checking for updated packages...
-No tags found! Comparing with initial commit.
+exports[`stderr: updates fixed versions 1`] = `
+"lerna info version __TEST_VERSION__
+lerna info current version 1.0.0
+lerna info Checking for updated packages...
+lerna WARN No tags found!
+lerna info Comparing with initial commit.
```

Comment:
I made the `No tags found!` part of the message an actual warning, so it jumps out. No functional change.